src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleRequestSender.cs

```diff
@@ -132,12 +136,14 @@ public async Task<int> InitializeCommunicationAsync(int clientConnectionTimeout)
         /// <inheritdoc/>
         public void InitializeExtensions(IEnumerable<string> pathToAdditionalExtensions)
         {
+            EqtTrace.Info("VsTestConsoleRequestSender.InitializeExtensions: Initializing extensions.");
```

Comment:
Shall we log the information as well ? Like pathToAdditionalExtensions

Reply:
Added